### DISCUSSION TITLE
Fold text on extremely long strings

### DISCUSSION THREAD

**Discussion by versecafe** [zed-industries/zed#23946] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

On long strings such as tailwind classes or large block sections for multiline strings being able to fold them up like this makes things far more readable.

### If applicable, add mockups / screenshots to help present your vision of the feature

<img width="336" alt="image" src="https://github.com/user-attachments/assets/a74f9080-fd2b-431a-a02a-f417e3dadc40">

**Comment by simply-zaki** [comment]
i agree, its quite painful to code with tailwind without folding
